enterprise/management/commands/assign_enterprise_user_roles.py

```diff
@@ -87,9 +105,27 @@ def _get_enterprise_customer_users_batch(self, start, end):
         Returns a batched queryset of EnterpriseCustomerUser objects.
         """
         LOGGER.info('Fetching new batch of enterprise customer users from indexes: %s to %s', start, end)
-        return User.objects.filter(pk__in=EnterpriseCustomerUser.objects.values('user_id'))[start:end]
+        return User.objects.filter(pk__in=self._get_enterprise_customer_user_ids())[start:end]
```

Comment:
nit: I think the previous one is more descriptive.

Reply:
yeah, it's used on a couple places so moved to a method